README.md

```diff
@@ -1,90 +1,89 @@
-
-## Merging PRs
-
-- PRs must be approved by at least one team member before they are merged.
-  - This can mean a delay of up to a day or two, because our team is spread across many time zones. Please be patient.
-  - Reach out to folks on Slack, either in the team channel or by DMs, if you're trying to accelerate the process.
-- Please do not merge your PR if you have 1 approval and there is a change requested by another approver. Please address all comments before merging the PR.
-- We encourage people to merge their own PRs after approval!
-  - Merge conflicts will happen sometimes and it's good to learn about resolving them.
-  - If you're uncomfortable doing your own merge, ask if someone can help walk you through it.
-  - If you know that another PR has touched the same file(s) as yours and got merged before yours, you can resolve the merge locally on your branch, pushing that change, and then merging the PR.
+## Features
+
+- Feature 1: This Project includes 4 fully responsive web page create with React
+- Feature 2: It includes [Hash Router](https://reactrouter.com/en/main/router-components/hash-router) for the implementation of the routing system
+- Feature 3: The Typography and Button across this project uses a common styling system
+- Feature 4: As part of the Innovation of the challenge, this website has an explicit error page for invalid routes.
+- Feature 5: The project uses css modules for the implementation of the stylesheet
+- Feature 6: The project implements a Form with Client side validation.
+- Feature 7: The project is deployed using Github Pages and Github Actions. [Deployed URL](https://poulami-saha.github.io/Global-Code-Queens-MyTeam-Website/#/)
+- Feature 8: This project uses npm as its package manager.
+
+## Installation
+
+To install and set up this project, follow these steps:
+
+### Prerequisites
+
+- The Project is build with React v18
+- The Project is handled with git and Github. Please follow the procedure linked [here](documents/githubLearning.md)
+
+### Clone the Repository
+
+```bash
+git clone git@github.com:poulami-saha/Global-Code-Queens-MyTeam-Website.git
+cd Global-Code-Queens-MyTeam-Website
+```
+
+### Install Dependencies
+
+```bash
+npm install
+```
+
+### Running the Application
+
+```bash
+npm start
+```
+
+### Repository Author & Maintainers
+
+1. [Rika Williams](https://github.com/berikawilliams)
+2. [Manami W](https://github.com/chachamana)
+3. [Eunice Y](https://github.com/eunicey)
+4. [Feven Seyfu](https://github.com/FevenSeyfu)
+5. [Joan Wolkerstorfer](https://github.com/joanwolk)
+6. [Kezia Baidoo](https://github.com/keziabaidoo)
+7. [Sehaj Wadhwa](https://github.com/sehajwadhwa)
+8. [Uzma Khan](https://github.com/Uzmakh)
+9. [Poulami Saha](https://github.com/poulami-saha)
+
+### Contributing
+
+We welcome contributions! Please follow these steps to contribute:
```

Comment:
I recommend a link or at least reference to the githubLearning document from here

Reply:
Otherwise, looks good to me!

Reply:
Added now